### PR TITLE
Imran/update travis

### PR DESCRIPTION
**Pull request by blakegong**
@imohammedzen please help take over this PR. Thank you!

@zendesk/basketcases

### CODE REVIEW

**Comment by blakegong** [comment]
I think the warning message commit could be dropped. With the latest repo settings, no one would be able to force push anymore, and everything must go through PRs to land into `master`. There's no more risk of anyone messing up `master`.

**Comment by arnavk** [comment]
💪

**Comment by blakegong** [comment]
Hello @arnavk 👋 

All Zopim repos are migrated into Zendesk org recently, private or public.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+PLEASE REBASE AND DONT USE MERGE
```
Comment (hyperair) [inline README.md]: What's this?

Comment (blakegong) [reply README.md]: I think back then, some old commits might have some secret that's been scrubbed. This was a reminder for people not to force push to `master` to bring back those commits.

I've already disabled force push & normal push on `master`, this shouldn't be a problem anymore.

**Comment by felixputera** [comment]
Succeeded by #28

**Pull request closed without merging**